web-jehlomat/src/Components/Form/PridatUzivateleForm.tsx

```diff
@@ -0,0 +1,157 @@
+import { FC } from 'react';
+import { Formik } from 'formik';
+import { useMediaQuery } from '@mui/material';
+import TextInput from '../Inputs/TextInput/TextInput';
+import { FormItemLabel, FormItemDescription } from '../../utils/typography';
+import { FormWrapper, FormItem } from '../Form/Form';
+import PrimaryButton from '../Buttons/PrimaryButton/PrimaryButton';
+import Box from '@mui/material/Box';
+import { useHistory } from 'react-router-dom';
+import { authorizedAPI } from '../../config/baseURL';
+import { AxiosResponse } from 'axios';
+import * as yup from 'yup';
+import styled from '@emotion/styled';
+import { primaryDark } from '../../utils/colors';
+import { media } from '../../utils/media';
+import { LINKS } from 'utils/links';
+
+interface Props {}
+
+interface Values {
+    email: string;
+}
+
+const validationSchema = yup.object({
+    email: yup.string().email('Vlož validní email').required('Email je povinné pole'),
+});
+
+const ModalFormWrapper = styled(FormWrapper)<{ mobile?: boolean }>`
+    ${props =>
+        props.mobile &&
+        `
```

Comment:
nitpick: klidně můžeš obalit toto s fcí css. Bude to pak barevné jako ostatní

```typescript
import { css } from '@emotion/react';

${props =>
        props.mobile &&
       css`
            ...
        `
```

btw. na styly pro telefon klidně používejme staré dobré media query